BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/BeeAlertErrorDialog.java

```diff
@@ -0,0 +1,60 @@
+package com.blueprint.foe.beetracker;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.TextView;
+
+import com.blueprint.foe.beetracker.Listeners.BeeAlertDialogListener;
+
+/**
+ * A generic dialog that should be modified for each type of dialog using the Bundle passed into
+ * onCreateDialog.
+ */
+public class BeeAlertErrorDialog extends DialogFragment {
+    public static final String ERROR_MESSAGE = "ERROR_MESSAGE";
```

Comment:
Consider renaming so it's clearer it's a key to retrieve arguments i.e. ERROR_MESSAGE_KEY